examples/send_super_stream.rs

```diff
@@ -0,0 +1,78 @@
+use rabbitmq_stream_client::error::StreamCreateError;
+use rabbitmq_stream_client::types::{
+    ByteCapacity, HashRoutingMurmurStrategy, Message, ResponseCode, RoutingStrategy,
+};
+use std::sync::atomic::{AtomicU32, Ordering};
+use std::sync::Arc;
+use tokio::sync::Notify;
+
+fn hash_strategy_value_extractor(message: &Message) -> String {
```

Comment:
@wolf4ood : In the other clients here we are returning a field in the application_properties like "id" instead of the message. I had some difficulties to convert "SimpleValue" of applications_properties to String